dashboard.el

```diff
@@ -86,6 +86,11 @@
   :group 'dashboard
   :type 'hook)
 
+(defcustom dashboard-before-initialize-hook nil
+  "Hook that is run before dashboard buffer is initialized."
+  :group 'dashboard
+  :type 'hook)
```

Comment:
Let’s put this above the after-initialize hook, so it makes a bit more sense.

Reply:
fixed